.circleci/config.yml

```diff
@@ -4,7 +4,7 @@ jobs:
     working_directory: /go/src/k8s.io/helm
     parallelism: 3
     docker:
-      - image: golang:1.8
+      - image: golang:1.9
```

Comment:
Now that 1.9.1 is out (with security fixes), we should use that version

Reply:
IIRC 1.x "official" images at dockerhub are always pointing at the latest patch release but I can make this more explicit if you'd prefer. :)

Reply:
Oh, I think you are right with that